.github/workflows/release-api-package.yml

```diff
@@ -0,0 +1,77 @@
+name: Release api package
+
+on:
+  workflow_dispatch:
```

Comment:
Do we consider any additional triggers? But at the moment it's fine to have manual trigger only

Reply:
We may consider other triggers after manually releasing stability. :)